invoice/src/test/java/org/killbill/billing/invoice/generator/TestInAdvanceBillingIntervalDetail.java

```diff
@@ -42,13 +42,13 @@ public void testCalculateFirstBillingCycleDate1() throws Exception {
         final LocalDate targetDate = new LocalDate();
         final int bcd = 17;
 
-        final BillingIntervalDetail annualBillingIntervalDetail = new BillingIntervalDetail(from, to, targetDate, bcd, BillingPeriod.ANNUAL, BillingMode.IN_ADVANCE);
+        final BillingIntervalDetail annualBillingIntervalDetail = new BillingIntervalDetail(from, to, targetDate, bcd, BillingPeriod.ANNUAL, BillingMode.IN_ADVANCE, null);
```

Comment:
`InArrearMode.DEFAULT` instead of `null` (just makes it easier to read).

Reply:
Fixed in https://github.com/killbill/killbill/pull/1824/commits/038cb86de4ae59f86d385a7b11e090e3274a09c8